content/doc/book/managing/nodes.adoc

```diff
@@ -45,10 +45,10 @@ clock time/sync and response time.
 A node is taken offline if any of these values
 go outside the configured threshold.
 
-The Jenkins controller itself runs on a special _built-in node_.
-It is possible to run agents and executors on this built-in node
-although this can degrade performance, reduce scalability of the Jenkins instance, and create serious security problems
-and is strongly discouraged, especially for production environments.
+The built-in node is a node inside the controller process, hence the name. 
+It can have executors, but it is not an agent. Agents and the built-in node are nodes. 
+All agents are nodes, but not all nodes are agents. Executors exist, are available or busy, 
+and can be added or removed, but they are not "run".
```

Comment:
```suggestion
Jenkins can use two kinds of nodes: _agents_ (descibed below) and the _built-in node_. The built-in node is a node inside the controller process.
Both agents and built-in node can be used to run tasks, though running tasks on built-in node is link:/book/security/controller-isolation/[discouraged because of security, performance and scalability reasons].
Node's ability to run tasks is determined by the number of executors configured for the node.
To disable running tasks on built-in node you can set the number of executors to 0.
```
I think the bit about performance/security implications of using built-in node is still valid and should not be removed. The sentence about executors was just an explanation of why "running an executor" was an incorrect phrase in the old version -- explaining relation between executors and nodes here may make sense though (

Reply:
Thank you for the suggestions.